common/containers/OnboardModal/index.tsx

```diff
@@ -180,7 +184,7 @@ class OnboardModal extends React.Component<Props, State> {
     localStorage.setItem(ONBOARD_LOCAL_STORAGE_KEY, String(nextSlideNum));
     this.props.incrementSlide();
     if (this.modal) {
-      this.modal.scrollContentToTop();
+      this.modal.modalBody.scrollContentToTop();
```

Comment:
Can you just bubble this up to `modal` by giving it a public function of the same name that calls `this.modalBody.scrollContentToTop`? I'd like to encapsulate as much of the modal changes as possible.